lib/tasks/assign_role_to_admin.rake

```diff
@@ -0,0 +1,5 @@
+task :assign_role_to_admin, [:email] => :environment do |t,arg|
+  User.where(email: arg[:email]).each do |user|
```

Comment:
why where?
why would we have multiple users with same email?

Reply:
done